pages/en/linux/linux-on-chromeos-faq.md

```diff
@@ -85,7 +85,7 @@ Yes! You'll probably need to install the relevant packages first for whatever co
 
 ## What architecture works on my system?
 
-Since everything is all native code execution, it depends on the device you have.
+Since everything is all machine code execution, it depends on the device you have.
```

Comment:
I would rewrite this. "The Linux container architecture will match the architecture of the device you have. You can find out what that is in two different ways:"